styles/main.css

```diff
@@ -10,7 +13,11 @@
   --body-background-color: #ffffff;
   --dark-white: #f7f7f7;
   --item-background-color: #c5c5c5;
-  --hover-color: #fb7171;
+  --hover-color: #ef4444;
+  --status-online: rgb(70, 168, 70);
```

Comment:
Try to use one color format instead of mixtures.

Reply:
done!